.claude/scripts/branch-op-lock.test.sh

```
@@ -0,0 +1,368 @@
+  branch_op_lock_release "$repo" "$t"
+) &
+cleaner_pid=$!
+
+# Wait until cleaner holds the lock.
+for _ in 1 2 3 4 5 6 7 8 9 10; do
+  [[ -d "$lockdir" ]] && break
+  sleep 0.2
+done
+
+add_rc=0
+BRANCH_OP_LOCK_TIMEOUT_SEC=10
```

Shellcheck (0.11.0)

[warning] BRANCH_OP_LOCK_TIMEOUT_SEC appears unused. Verify use (or export if used externally).

(SC2034)